app/views/steps/check/check_your_answers/shared/_check_row.html.erb

```diff
@@ -1,6 +1,15 @@
-<h3 class="govuk-heading-m govuk-!-margin-bottom-2">
-  <%=t "#{check_row.kind}.orders.#{check_row.order_type}", scope: check_row.scope %>
-</h3>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
+      <%=t "#{check_row.kind}.orders.#{check_row.order_type}", scope: check_row.scope %>
+    </h3>
+  </div>
+
+  <div class="govuk-grid-column-one-half app-text-align-right">
+    <%= link_to('Remove this sentence', edit_steps_check_path(check_row.disclosure_check), class: 'govuk-link govuk-link--no-visited-state app-link-red govuk-!-font-size-19') %>
```

Comment:
Upps, just realised, for cautions this might need to read "Remove this caution" and for convictions not everything is a sentence so again we have the problem of naming things 🤣 Wonder if "Remove this check" is good enough 😞

Reply:
maybe "Remove this one" / "Remove this"